packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -138,9 +138,77 @@ class _DriverBinding extends BindingBase with SchedulerBinding, ServicesBinding,
 /// }
 /// ```
 ///
-void enableFlutterDriverExtension({ DataHandler? handler, bool silenceErrors = false, List<FinderExtension>? finders}) {
+/// Note: SomeCommand, SomeResult and SomeCommandExtension should be placed in
+/// different files to avoid `dart:ui` import issue.
```

Comment:
nit: explain why you can't import dart:ui for the host side test script.

Reply:
Done.